### PR TITLE
Removed the border-radius on align left

### PR DESCRIPTION
**Pull request by Spoorthy1423**
#741 Removed border-radius on align left
Removed unused sections and

Removed border-radius when the thumbnail display is align left using thumbnailAlign prop and made thumbnail prop large.

#### Issue addressed

Addresses #*PR# HERE*

### Before/after screenshots
Screenshot 2024-08-24 at 6 43 33 PM

## Changelog

- [#PR no]
  - **Description:** Summary of change(s)
  - **Products impact:** Choose from - none (for internal updates) / bugfix / new API / updated API / removed API. If it's 'none', use "-" for all items below to indicate they are not relevant.
  - **Addresses:** Link(s) to GH issue(s) addressed. Include KDS links as well as links to related issues in a consumer product repository too.
  - **Components:** Affected public KDS component. Do not include internal sub-components or documentation components.
  - **Breaking:** Will this change break something in a consumer? Choose from: yes / no
  - **Impacts a11y:** Does this change improve a11y or adds new features that can be used to improve it? Choose from: yes / no
  - **Guidance:** Why and how to introduce this update to a consumer? Required for breaking changes, appreciated for changes with a11y impact, and welcomed for non-breaking changes when relevant.

[#PR no]: PR link

## Steps to test

1. Step 1
2. Step 2
3. ...

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change is described in the changelog section above

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

**Comment by MisRob** [comment]
@Spoorthy1423 Thank you - if you can, it'd be a bit easier if you just pushed new commits to already open pull request rather than closing after every feedback, so we can follow what happened and why. I will loop in @AllanOXDi for review here - Allan, also please note https://github.com/learningequality/kolibri-design-system/pull/753#issuecomment-2309609785

**Comment by Spoorthy1423** [comment]
Hey!! @AllanOXDi can u please review my pr , today is the deadline

**Comment by MisRob** [comment]
@Spoorthy1423 please be thoughtful towards the community and respect https://github.com/learningequality/kolibri-design-system/discussions/751#discussioncomment-10443675.

File: lib/KCard/index.vue [hunk 1]
```diff
@@ -320,7 +320,7 @@
             },
             thumbnailStyles: {
               ...thumbnailCommonStyles,
-              borderRadius: '8px 0 0 8px',
+              borderRadius: '0 8px 8px 0',
```
Comment (AllanOXDi) [inline lib/KCard/index.vue]: Hi @Spoorthy1423 . Thanks for you work here. I have reviewed it and I think it's still breaking when it's align left.
Screenshot 2024-08-26 at 16 05 16

Comment (AllanOXDi) [reply lib/KCard/index.vue]: I guess we will need to swap it  when the thumbnail is on the right
`borderRadius: '0 8px 8px 0',`

And when the thumbnail is on the left
`borderRadius: '8px 0 0 8px',`

Comment (AllanOXDi) [reply lib/KCard/index.vue]: Once we confirm that everything is looking good we will go a head and merge it.

**Review by AllanOXDi — COMMENTED** [review]
Hi @Spoorthy1423  Thanks for updating the logic . I just want to note that the linting check is failing. Could you run pre-commit as described in [this section of the developer documentation](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#manual-linting-and-formatting)? and commit it again.
I will let my colleague @MisRob  to give it a second eye and we merge

**Comment by Spoorthy1423** [comment]
@AllanOXDi i am actually facing this issue 

(myenv) spoorthymadasu@spoorthys-MacBook-Air kolibri-design-system % pre-commit run --all-files
An error has occurred: InvalidConfigError: 
=====> .pre-commit-config.yaml is not a file
Check the log at /Users/spoorthymadasu/.cache/pre-commit/pre-commit.log

do i need to create a pre-commit-config.yaml

**Comment by AllanOXDi** [comment]
Sorry could you try `yarn lint -w`

**Comment by AllanOXDi** [comment]
I guess you will have to do` pre-commit install `first

**Comment by AllanOXDi** [comment]
Hi @Spoorthy1423  I am going to reopen this PR and close the one you just opened. Reason being that this  one already have the reviews

**Comment by AllanOXDi** [comment]
As for the failing lint check. I have just fixed that and it should pass. I will wait for my colleague to give it a final look and we will proceed to marge it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Upcoming version 5.x.x (`develop` branch)
 
+- [#741]
+  - **Description:** Removed the border-radius on align left.
```
Comment (MisRob) [inline CHANGELOG.md:8]:
```suggestion
  - **Description:** Fixed the border-radius on align left.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
...
+  - **Products impact:** -.
+  - **Addresses:** -.
+  - **Components:** -.
+  - **Breaking:** no -.
+  - **Impacts a11y:** no -.
+  - **Guidance:** - Removed the border-radius on align left.
```
Comment (MisRob) [inline CHANGELOG.md]:
```suggestion
  - **Guidance:** -
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Upcoming version 5.x.x (`develop` branch)
 
+- [#741]
+  - **Description:** Removed the border-radius on align left.
+  - **Products impact:** -.
```
Comment (MisRob) [inline CHANGELOG.md]:
```suggestion
  - **Products impact:** bugfix.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
...
+  - **Addresses:** -.
```
Comment (MisRob) [inline CHANGELOG.md]:
```suggestion
  - **Addresses:** https://github.com/learningequality/kolibri-design-system/issues/741
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
...
+  - **Components:** -.
```
Comment (MisRob) [inline CHANGELOG.md]:
```suggestion
  - **Components:** `KCard`
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Upcoming version 5.x.x (`develop` branch)
 
+- [#741]
```
Comment (MisRob) [inline CHANGELOG.md]:
```suggestion
- [#754]
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
...
+  - **Description:** Removed the border-radius on align left.
+  - **Products impact:** -.
+  - **Addresses:** -.
+  - **Components:** -.
+  - **Breaking:** no -.
+  - **Impacts a11y:** no -.
+  - **Guidance:** - Removed the border-radius on align left.
+
+[#741]: https://github.com/learningequality/kolibri-design-system/pull/751
```
Comment (MisRob) [inline CHANGELOG.md:16]:
```suggestion
[#754]: https://github.com/learningequality/kolibri-design-system/pull/754
```

**Comment by MisRob** [comment]
Thanks both. Just few changelog tweaks to help the release manager and should be good to go.

**Comment by Spoorthy1423** [comment]
Hey!! both tqsm for letting me know we can commit change directly , if anything else needed, can u please let me know, i got some idea on how to make this work - tqsm .

**Review by AllanOXDi — APPROVED** [review]

**Comment by MisRob** [comment]
Thanks both here. All looks good to me now. I will merge it now and it will be a part of today's release.

@Spoorthy1423 If part of the reason for so many prs was working with git, this is said to be a fun way to learn more https://learngitbranching.js.org/. Also, I usually recommend authors to preview  "Files changed" tab here in GitHub UI to be sure that there are no changes that were not supposed to be there, ideally before marking the PR as ready for review. I really don't know much about you - just my best guesses of what may perhaps help to contribute smoothly.

**Comment by Spoorthy1423** [comment]
Thank you both for your patience in reviewing my multiple PRs. @MisRob, I will certainly follow your suggestions. The insights and knowledge I gained from this PR are invaluable. Thank you very much.

**Comment by MisRob** [comment]
Thanks for saying that - @AllanOXDi did most of the work (I'm just sticking my nose into things and commenting :) Good luck in your endeavors.

**Pull request merged**